openmc/deplete/results.py

```diff
@@ -245,6 +246,10 @@ def _write_hdf5_metadata(self, handle):
         handle.create_dataset("power", (1, n_stages), maxshape=(None, n_stages),
                               dtype='float64')
 
+        handle.create_dataset(
+            "depletion time", (1, 1), maxshape=(None, 1),
```

Comment:
Since there's really only a single dimension, should this just be
```suggestion
            "depletion time", (1,), maxshape=(None,),
```